src/api/Jalasoft.TeamUp.Projects.API/Controllers/PostInvitation.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Jalasoft.TeamUp.Projects.API.Controllers
+{
+    using System;
+    using System.IO;
+    using System.Net;
+    using System.Threading.Tasks;
+    using FluentValidation;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Jalasoft.TeamUp.Projects.ProjectsException;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.OpenApi.Models;
+    using Newtonsoft.Json;
+
+    public class PostInvitation
+    {
+        private readonly IInvitationsService invitationService;
+
+        public PostInvitation(IInvitationsService invitationService)
+        {
+            this.invitationService = invitationService;
+        }
+
+        [FunctionName("PostInvitationByProjectId")]
```

Comment:
PostInvitation should be enough, the ProjectId is not sent as a parameter so is shouldn't be considered